tests/hypervisor/test_default.py

```diff
@@ -10,14 +10,15 @@
 
 from virtwho import REGISTER
 from virtwho import HYPERVISOR
+from virtwho import logger
 
 from virtwho.base import hostname_get
 from virtwho.configure import hypervisor_create
 
 
 @pytest.mark.usefixtures("function_host_register_for_local_mode")
 @pytest.mark.usefixtures("function_virtwho_d_conf_clean")
-@pytest.mark.usefixtures("class_debug_true")
+@pytest.mark.usefixtures("function_debug_true")
```

Comment:
Curious about this change, would it be something wrong when we using `"class_debug_true"` here?